clusterloader2/testing/density/constraints_100.yaml

```diff
@@ -0,0 +1,24 @@
+coredns:
+  cpuConstraint: 0.01
+  memoryConstraint: 9437184 #9 * (1024 * 1024)
+fluentd-gcp:
+  cpuConstraint: 1
+  memoryConstraint: 220200960 #210 * (1024 * 1024)
+heapster:
+  cpuConstraint: 0.1
+  nemoryConstraint: 340787200 #325 * (1024 * 1024)
+kube-apiserver:
+  cpuConstraint: 1.7
+  memoryConstraint: 975175680 #930 * (1024 * 1024)
+kube-controller-manager:
+  cpuConstraint: 0.9
+  memoryConstraint: 241172480 #230 * (1024 * 1024)
+kube-proxy:
+  cpuConstraint: 0.01
+  memoryConstraint: 10485760 #10 * (1024 * 1024)
+kube-scheduler:
+  cpuConstraint: 0.22
+  memoryConstraint: 94371840 #90 * (1024 * 1024)
+l7-lb-controller:
+  cpuConstraint: 0.08
+  memoryConstraint: 65011712 #62 * (1024 * 1024)
```

Comment:
70 maybe? Let's not make them too artificial (62 seem too artificial to me).

Reply:
Changed to 80.